actions/actions.py

```diff
@@ -549,3 +551,17 @@ async def run(
         events.append(ActionExecuted("action_listen"))
 
         return events
+
+
+class ActionRestart(Action):
+    def name(self) -> Text:
+        return "action_restart"
+
+    async def run(
+        self,
+        dispatcher: CollectingDispatcher,
+        tracker: Tracker,
+        domain: Dict[Text, Any],
+    ) -> List[EventType]:
+
+        return [Restarted(),FollowupAction("action_session_start")]
```

Comment:
Code linting throwing two errors: missing a space after the comma and missing a newline after the return call